tests/users/searchUser.test.ts

```diff
@@ -0,0 +1,54 @@
+import request from 'supertest';
+import { expect } from 'chai';
+import app from '../../src/app';
+import { sandbox } from '../setup';
+import * as responses from '../../src/utils/responses';
+import database from '../../src/services/database';
+import { Error } from '../../src/types';
+import { createFakeUser } from '../utils';
+
+describe('GET /users/search', () => {
+  before(async () => {
+    await createFakeUser({
+      username: 'rinkichi',
+      email: 'ragequit@ez.com',
+    });
+  });
+
+  after(async () => {
+    await database.user.deleteMany();
+  });
+
+  it('should fail with an internal server error', async () => {
+    sandbox.stub(responses, 'success').throws('Unexpected error');
+
+    await request(app).get('/users/search?query=rinkichi').expect(500, { error: Error.InternalServerError });
+  });
+
+  it('should not accept empty query parameters', async () => {
+    await request(app).get('/users/search?query=').expect(400, { error: Error.InvalidQueryParameters });
+  });
+
+  it('should not accept bad query parameters', async () => {
+    await request(app).get('/users/search?query=').expect(400, { error: Error.InvalidQueryParameters });
```

Comment:
J'avais pas vu que c'était exactement la même chose qu'au dessus, la modif de `query` n'a pas du passer 😉
```suggestion
    await request(app).get('/users/search?find=').expect(400, { error: Error.InvalidQueryParameters });
```

Reply:
Pourquoi ça ?

Reply:
C'est exactement la même ligne que la ligne 29, j'imagine que le but de ce test (au vu de son nom) est de vérifier le nom des params de la query

Reply:
Ah oui, en effet